Cython/Compiler/FlowControl.py

```diff
@@ -943,6 +944,11 @@ def mark_forloop_target(self, node):
         is_special = False
         sequence = node.iterator.sequence
         target = node.target
+        if (isinstance(sequence, UtilNodes.ComprehensionResultRefNode) and
+            sequence.is_main_result):
```

Comment:
An additional overindentation here would make it clearer to spot that this line is still part of the condition and not the first statement in the if-block. (Same in other places)